bndtools.core/src/bndtools/wizards/bndfile/NewBndFileWizardPage.java

```diff
@@ -57,8 +57,14 @@ protected boolean validatePage() {
         IResource container = ResourcesPlugin.getWorkspace().getRoot().findMember(containerPath);
         IProject project = container.getProject();
 
-        if (Project.BNDFILE.equalsIgnoreCase(fileName)) {
-            error = Messages.NewBndFileWizardPage_errorReservedFilename;
+        try {
+            if (project.hasNature(BndtoolsConstants.NATURE_ID)) {
+                if (Project.BNDFILE.equalsIgnoreCase(fileName)) {
+                    error = Messages.NewBndFileWizardPage_errorReservedFilename;
```

Comment:
@gamerson, this is what I see in latest:

![image](https://cloud.githubusercontent.com/assets/114601/18323398/689265ba-7505-11e6-9b28-ed5f8ebddade.png)